app/redux-modules/feedback.js

```diff
@@ -51,14 +54,28 @@ export default function reducer(state = initialNavState, action) {
       return Object.assign({}, state, feedback);
     }
     case CREATE_FEEDBACK: {
-      const reports = { ...state.list, ...action.payload };
-      return Object.assign({}, state, { list: reports });
+      const feedback = { ...state.list, ...action.payload };
+      return Object.assign({}, state, { list: feedback });
     }
     case UPDATE_FEEDBACK: {
       const list = Object.assign({}, state.list);
       list[action.payload.name] = Object.assign({}, state.list[action.payload.name], action.payload.data);
       return Object.assign({}, state, { list });
     }
+    case UPLOAD_FEEDBACK_REQUEST: {
+      const { name, status } = action.payload;
```

Comment:
To avoid crashes while the report is being uploaded can we add it to the sync flow? so when the report is uploading and we are online we can show the syncing modal 😬